docs/docs/reference-dom-elements.md

```diff
@@ -29,31 +98,18 @@ select small source span strong style sub summary sup table tbody td textarea
 tfoot th thead time title tr track u ul var video wbr
 ```
 
-#### SVG elements
+## All Supported SVG Elements
```

Comment:
This is incorrect. The list below is the list of SVG elements for which we have shortcuts in discouraged React.DOM factories.

However if you use JSX (of if you pass strings to createElement and avoid React.DOM factories), **all SVG elements are supported out of the box**. There is no whitelist for them.